lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -672,8 +676,12 @@ def add_po_number(xml, options)
         xml.poNumber(options[:po_number]) if options[:po_number]
       end
 
+      def add_extra_options_for_cim(xml, options)
+        xml.extraOptions("x_delim_char=#{options[:delimiter]}") if options[:delimiter]
```

Comment:
Is this always legal to add? I'm asking because, if it is, we could fold it directly into `commit`. (I'm aware that it's not currently added for a lot of the customer profile calls, hence the question.)

Reply:
I'll check.

Reply:
Unfortunately no, this is probably safer.

Reply:
No worries! Thanks for checking.